core/src/avm2/globals/flash/ui/MouseCursorData.as

```diff
@@ -0,0 +1,26 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/ui/MouseCursorData.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.ui
+{
+    import flash.geom.Point;
+    
+    public final class MouseCursorData
+    {
+        // A Vector of BitmapData objects containing the cursor image or images.
+        public var data: Vector;
```

Comment:
I think this should be `Vector.<BitmapData>`.

Reply:
Thank you!